plugin.js

```diff
@@ -209,13 +209,22 @@ export default plugin(
         'background-color': 'currentColor',
       },
       [`[type='file']`]: {
-        background: 'unset',
-        'border-color': 'inherit',
-        'border-width': '0',
-        'border-radius': '0',
-        padding: '0',
-        'font-size': 'unset',
-        'line-height': 'inherit',
+        display: 'block',
+        width: '100%',
+        cursor: 'pointer',
+        overflow: 'hidden',
+        'border-radius': '.5rem',
```

Comment:
Hi and thanks for this PR.

The border-radius seems off to me by taking a look at the screenshot, and it's confirmed because of `.375rem` in the other buttons

Can you please use the settings from the theme?

I can see 

```
        'border-width': borderWidth['DEFAULT'],
        'border-radius': borderRadius.none,
```

in this file. Don't know exactly how. flowbite works. It looks like it is using `rounded-md` class

Reply:
@tagliala Thanks, done! I've updated the PR to use theme config as much as possible. I reviewed the file again and I've realised I could replace all those styles with just one big `@apply`:

```
[`[type='file']`]: {
  @apply block w-full text-sm text-gray-900 border border-gray-300 rounded-lg cursor-pointer bg-gray-50 dark:text-gray-400 focus:outline-none dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 overflow-hidden;
}
```

Do you have preferred style? I've seen both inlines and applies in this file.

Reply:
Thanks

> Do you have preferred style?

Can't tell exactly, not my expertise, let's wait for @javierjulio 's review, I guess it should be a mix of both but I can't tell exactly what should be in a class and what in that definition